src/main.cpp

```diff
@@ -203,16 +228,73 @@ void rpcWSWrapper() {
 void GetDeviceInfo() {
   JsonDocument jsonResponse;
   jsonResponse["name"] = shelly_name;
-  jsonResponse["id"] = shelly_name;
+  jsonResponse["id"] = shelly_name; // ???? ToDo check
   jsonResponse["mac"] = shelly_mac;
-  jsonResponse["slot"] = 1;
-  jsonResponse["model"] = "SPEM-003CEBEU";
-  jsonResponse["gen"] = 2;
-  jsonResponse["fw_id"] = "20241011-114455/1.4.4-g6d2a586";
-  jsonResponse["ver"] = "1.4.4";
-  jsonResponse["app"] = "Pro3EM";
+  jsonResponse["slot"] = shelly_slot;
+  jsonResponse["model"] = shelly_model;
+  jsonResponse["gen"] = shelly_gen;
+  jsonResponse["fw_id"] = shelly_fw_id;
+  jsonResponse["ver"] = shelly_ver;
+  jsonResponse["app"] = shelly_app;
   jsonResponse["auth_en"] = false;
-  jsonResponse["profile"] = "triphase";
+  jsonResponse["auth_domain"] = nullptr;
+  jsonResponse["profile"] = power_variant;
+  serializeJson(jsonResponse,serJsonResponse);
+  DEBUG_SERIAL.println(serJsonResponse);
+}
+
+void EM1GetStatus_mono(){
+  JsonDocument jsonResponse;
+  // Reconstruction structure for FHEM -> no WARNINGS 
+  jsonResponse["id"] = 0;
+  jsonResponse["current"] = 2.22;

```

Comment:
dann in diesen Zeilen aber auch die echten werte anbieten und keine Fakes

Reply:
Das kommt natürlich noch herein :-) Um auch die Auswertung zu sehen ohne Anbindung zu Geräten, wurde es nur für Testzwecke hart verankert.